.github/workflows/release.yml

```diff
@@ -0,0 +1,65 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - '*'
+
+permissions:
+  contents: write
+
+jobs:
+  publish_derive:
+    name: Publish
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    if: github.ref_type == 'tag'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Install stable
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+
+      - name: Publish
+        env:
+          TOKEN: ${{ secrets.CRATES_IO_TOKEN }}
+        run: |
+          cargo login $TOKEN
+          cargo publish --allow-dirty -p ublox_derive
```

Comment:
Why is `--allow-dirty` needed?

Reply:
In practice it is not, I copied from somewhere else.
Publication might fail for example, if your CI generates stuff that is not _gitignored_, the branch becomes "dirty" and publication is disabled. To avoid this, I simply put it all the time, I consider that tagging demonstrates that CI and release should be obtained